src/settings.tsx

```diff
@@ -18,9 +20,25 @@ export const allLanguages: Language[] = [
     id: "en",
     name: "English",
     enName: "English",
+    makeCode: true,
+  },
+  {
+    id: "ja",
+    name: "日本語",
+    enName: "Japanese",
+    makeCode: true,
+  },
+  {
+    id: "ko",
+    name: "한국어",
+    enName: "Korean",
+    makeCode: true,
   },
```

Comment:
These additional languages should be removed for this PR. A subsequent PR will likely need something similar to classroom where we give the status on language/translations for the app and MakeCode (and the PythonEditor).

Reply:
https://github.com/microbit-foundation/ml-trainer/pull/290/commits/6597e213dea0f06e06e0978d40016ccad78fa305